stoobly_agent/app/cli/scaffold/validate_command.py

```diff
@@ -24,20 +33,28 @@ def __get_container(self, container_name: str) -> Container:
       except docker_errors.NotFound:
         sleep(0.5)
 
-    raise ScaffoldValidateException(f"Container not found: {container_name}")
+    error_message = self.__generate_container_not_found_error(container_name)
+    raise ScaffoldValidateException(error_message)
 
   def validate_init_containers(self, init_container_name, configure_container_name) -> None:
     print(f"Validating setup containers: {init_container_name}, {configure_container_name}")
 
-
-    init_container = self.__get_container(init_container_name)
+    init_container = self.__get_container_with_retries(init_container_name)
     logs = init_container.logs()
+
     if logs and re.search('error', str(logs), re.IGNORECASE):
-      raise ScaffoldValidateException(f"Error logs potentially detected in: {init_container_name}")
+      error_found_message = f"{bcolors.FAIL}Error logs potentially detected in: {init_container_name}{bcolors.ENDC}"
+      suggestion_message = f"{bcolors.BOLD}Run 'docker logs {init_container_name}'. Report a bug at https://github.com/Stoobly/stoobly-agent/issues{bcolors.ENDC}"
```

Comment:
User defined commands can also potentially generate errors, we shouldn't be recommending to file a bug here. We should instead also recommend checking the init script.